src/readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: wordpressdotorg
 Donate link: https://wordpressfoundation.org/donate/
 Tags: importer, wordpress
 Requires at least: 5.2
-Tested up to: 6.4.2
+Tested up to: 6.6.2
```

Comment:
```suggestion
Tested up to: 6.6
```

The Tested up to header should only contain a X.Y version number, it's not required to include the minor version.

Reply:
Tested, no problem with 6.7-beta2